Makefile

```diff
@@ -577,9 +577,6 @@ install-manpages: ## Install manpages the Cilium CLI.
 postcheck: build ## Run Cilium build postcheck (update-cmdref, build documentation etc.).
 	$(QUIET)$(MAKE) $(SUBMAKEOPTS) -C Documentation update-cmdref check
 
-minikube: ## Setup a minikube with Cilium installed.
```

Comment:
One less make target that has nothing to do with builds, yet does too much - yey! 🎉 

(This probably needs a separate commit, not sure how it relates to docs changes).

Reply:
I guess it happen to be an incidental dependency of quick install generation logic?

Reply:
It wasn't a dependency of quick install generation logic, but rather a target similar to `microk8s` - it builds cilium container image and then deploys it on minikube. My goal was to have some kind of lightweight alternative to vagrant.

But since it didn't end up being used, I'm fine with removing it. There is still microk8s for similar workflow. And running locally built images on kind is also an option.